csi.proto

```diff
@@ -420,6 +420,18 @@ message VolumeCapability {
       // Can be published as read/write at multiple nodes
       // simultaneously.
       MULTI_NODE_MULTI_WRITER = 5;
+
+      // Can only be published once as read/write at a single workload
+      // on a single node, at any given time. SHOULD be used instead of
+      // SINGLE_NODE_WRITER for COs using the experimental
+      // SINGLE_NODE_MULTI_WRITER capability.
```

Comment:
@chrishenzie this should be `SINGLE_NODE_SINGLE_WRITER` ?

Reply:
Both `SINGLE_NODE_SINGLE_WRITER` and `SINGLE_NODE_MULTI_WRITER` are replacements for `SINGLE_NODE_WRITER` depending on the number of intended writers. The capability `SINGLE_NODE_MULTI_WRITER` implies support for `SINGLE_NODE_MULTI_WRITER` and/or `SINGLE_NODE_SINGLE_WRITER`. The reason for naming the capability like so was because it was the access mode that was missing from the CSI spec.

Reply:
`The capability SINGLE_NODE_MULTI_WRITER implies support for SINGLE_NODE_MULTI_WRITER and/or SINGLE_NODE_SINGLE_WRITER` -> yeah, this aligns with my thought too or what I meant with `replacement as a default` in above comment, so we are good to mention this over over here ! Thanks @chrishenzie. :+1: